.github/workflows/flutter_package.yml

```diff
@@ -47,6 +47,10 @@ on:
         required: false
         type: string
         default: "."
+      package_get_excludes:
```

Comment:
As you said, the fact that we don't get the dependencies for a package doesn't mean that the package wont be tested. 

Afaik right now the user can:
- Avoid getting the package dependency with `package_get_excludes`
- Avoid collecting coverage for a package with `coverage_excludes`

I like the suggestion with `packages_excludes` or `packages_ignore`. I'm not sure if it is too generic, I can't think of use-cases where a user doesn't want to get the dependencies for a package and still test the package.  In the case that such use-case exists maybe we can add a `test_excludes` or `test_ignores` input to resolve what you mentioned here. However, since I don't think a use-case exists I like the generic name. Maybe the original author of the issue, @felangel might have such use-case.

Wdyt? Shall we go forward with `packages_ignore`?

Reply:
@wolfenrain I think @felangel use-case is valid and we should keep as is.